client/jetpack-connect/happychat-button.jsx

```diff
@@ -46,7 +54,7 @@ const JetpackConnectHappychatButton = ( {
 		<HappychatButton
 			borderless={ false }
 			className="logged-out-form__link-item jetpack-connect__happychat-button"
-			onClick={ onClick }
+			onClick={ getHappyChatButtonClickHandler( eventName ) }
```

Comment:
I think you'll still need to refactor this one as a ES6 class. This line overrides the `onClick` prop, and sometimes we want to specifically pass one to `JetpackConnectHappychatButton`, like we do here: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/site-settings/disconnect-site/troubleshoot.jsx#L31 - probably it makes sense to keep it and call it in `getHappyChatButtonClickHandler` as well.

Reply:
Oh, damn it, I didn't include that file in the last commit... I changed that instance to normalize the way we trigger events (that 'onclick' was being used just to trigger a tracks event):

https://github.com/Automattic/wp-calypso/pull/19181/files#diff-1ea929ab05c5c3de5c3c02bdb660aa4eL31